### DISCUSSION TITLE
Questions regarding KDE Plasmoids and Tiling Window Managers

### DISCUSSION THREAD

**Discussion by YamiYukiSenpai** [lxqt/lxqt#2045] [Q&A] [answered]
* Do KDE Plasmoids, or any plasmoids made for KDE work with LXQt?
  * I was informed [here](https://www.reddit.com/r/LXQt/comments/ort2pv/questions_regarding_kde_plasmoids_and_tiling/h6kndhx?utm_source=share&utm_medium=web2x&context=3), that Plasmoids won't work with LXQt's panel, so Latte should run on it?
* Which tiling window manager would you recommend with LXQt?
  * I just discovered that opening a new window on the desktop (at least on i3) will open in half-tiled because the desktop itself is PCManFM-Qt.  How can I make that instance of PCManFM-Qt to be "normal"?  I tried to make it float, but it'll always be on top of other windows, and I want it always below.

**Comment by tsujan** [comment, marked as answer]
> Do KDE Plasmoids, or any plasmoids made for KDE work with LXQt?

"Plasma" and "Plasmoid" are specific to KDE. Something like Latte-Dock is integrated into KDE. So, the answer is either "no" or "not without problem."

> I just discovered that opening a new window on the desktop (at least on i3) will open in half-tiled...

Under X11, that's a bug in the WM; under Wayland, it's a shortcoming of Wayland.

**Reply by YamiYukiSenpai** [reply]
> Under X11, that's a bug in the WM

Is PCManFM-Qt needed to draw the background?  I don't need the desktop icons.

**Reply by tsujan** [reply]
Stop and uncheck `LXQt Session Settings → Basic Settings → Desktop`.

**Reply by stefonarch** [reply]
> I don't need the desktop icons.

You can also disable them, right click, don't show desktop items

**Reply by YamiYukiSenpai** [reply]
> You can also disable them, right click, don't show desktop items

Main reason was that I needed the tiling to work properly.  The Desktop itself was tiling, which I was trying to fix.

I got it to work!  Thanks.  And I'm using Nitrogen to draw wallpaper.  If you have a better option that can potentially integrate with LXQt, I'm all ears.
